plugin/src/ChangeBatcher/encodePatchUpdate.spec.lua

```diff
@@ -37,6 +37,18 @@ return function()
 		expect(update.changedName).to.equal("We'reGettingToTheCoolPart")
 	end)
 
+	it("should recreate instance in the update when the instance's ClassName changes", function()
+		local part = Instance.new("Part")
+		local properties = {
+			ClassName = true
```

Comment:
We should set this value to a real `ClassName` so that future validation in the patch update function won't choke when it sees a bool instead.